specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,2059 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "LogicAppsManagementClient",
+        "version": "2016-06-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
```

Comment:
we only support application/json (in fact ARM would not even proxy text/json requests)

Reply:
That was pulled from the original file, so I left it alone, even though it was throwing a warning. I'll remove it